.github/workflows/deploy_heroku.yml

```diff
@@ -53,7 +53,7 @@ jobs:
           password: ${{ env.DOCKER_REGISTRY_TOKEN }}
 
       - name: Build Docker image
-        run: bin/docker-prepare && docker compose build
+        run: bin/docker-prepare && docker compose build --no-cache
```

Comment:
Why is `--no-cache` required?

Reply:
:monk I misunderstood since building the image on locally with no cache for testing, on Heroku deployment we don't need this option 🙏